src/nextjournal/clerk/sci_viewer.cljs

```diff
@@ -188,19 +188,17 @@
         [:div.flex-auto.h-screen.overflow-y-auto
          {:ref ref-fn}
          (into [:div.flex.flex-col.items-center.viewer-notebook.flex-auto]
-               (map (fn [x]
-                      (let [viewer (viewer/viewer x)
-                            blob-id (:blob-id (viewer/value x))
+               (map (fn [[_inspect x :as y]]
+                      (let [{viewer-name :name} (viewer/viewer x)
                             inner-viewer-name (some-> x viewer/value viewer/viewer :name)]
                         [:div {:class ["viewer" "overflow-x-auto"
-                                       (when (keyword? viewer) (str "viewer-" (name viewer)))
+                                       (when viewer-name (str "viewer-" (name viewer-name)))
                                        (when inner-viewer-name (str "viewer-" (name inner-viewer-name)))
-                                       (case (or (viewer/width x) (case viewer (:code :code-folded) :wide :prose))
+                                       (case (or (viewer/width x) (case viewer-name (:code :code-folded) :wide :prose))
                                          :wide "w-full max-w-wide"
                                          :full "w-full"
                                          "w-full max-w-prose px-8")]}
-                         (cond-> [inspect x]
-                           blob-id (with-meta {:key blob-id}))])))
+                         y])))
```

Comment:
Here blocks can be inlined as they come, since with the top-level `fetch-all` all wrapped value are inspected. Was surprised that this just works, but didn't see any troubles so far.